fuzz/cmd/compare/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	ds "github.com/ipfs/go-datastore"
+	fuzzer "github.com/ipfs/go-datastore/fuzz"
+	dsq "github.com/ipfs/go-datastore/query"
+
+	"github.com/spf13/pflag"
```

Comment:
I'd slightly prefer to use github.com/urfave/cli to be consistent with the other projects.

Reply:
Given that i'm not pulling in a full app/cli framework for these entry points, i'm going to defer on that for now.